pkg/util/validation/limits.go

```diff
@@ -1,13 +1,18 @@
 package validation
 
 import (
+	"errors"
 	"flag"
 	"os"
 	"time"
 
 	"gopkg.in/yaml.v2"
 )
 
+const (
```

Comment:
A common pattern is to just declare the error here:
`var errMaxGlobalSeriesPerUserValidation = errors.New(test)`